packages/dom/src/serialize-cssom.js

```diff
@@ -34,6 +42,47 @@ export function serializeCSSOM({ dom, clone }) {
       cloneOwnerNode.remove();
     }
   }
+
+  // clone stylesheets in shadowRoot
+  // https://github.com/WICG/construct-stylesheets/issues/93
+  if (!adoptedStylesMap) {
+    adoptedStylesMap = new Map();
+  }
+
+  for (let sheet of dom.adoptedStyleSheets) {
+    const styleLink = document.createElement('link');
+    styleLink.setAttribute('rel', 'stylesheet');
+
+    if (!adoptedStylesMap.has(sheet)) {
+      const styles = Array.from(sheet.cssRules)
+        .map(cssRule => cssRule.cssText).join('\n');
+      let resource = resourceFromText(uid(), 'text/css', styles);
+      resources.add(resource);
+      adoptedStylesMap.set(sheet, resource.url);
+    }
+    styleLink.setAttribute('href', adoptedStylesMap.get(sheet));
```

Comment:
Lets use 'data-percy-serialized-attribute-href' to avoid making network request (check video/canvas serialization)

Reply:
Nice!